### PR TITLE
docs: add Assignedtask with its description and constraints

### PR DESCRIPTION
**Pull request by yuki-92c**
## Overview

- I added the descriptions and constraints of `AssignedTask`

## Changes

> ## AssignedTask - 割当タスク
> 
> ### taskId
> 
> - TaskId is the UUID
> 
> ### title
> 
> - The owner can set a title as they prefer. This title can be modified at any time.
> 
> Constraints:
> 
> - It must be between 1 and 20 characters in length.
> 
> ### description
> 
> - The owner can provide details for the task. This description can be modified at any time.
> 
> Constraints:
> 
> - The description must be between 10 and 120 characters.
> 
> ### completed
> 
> - Whether the task is completed or not.

## Assignee Checklist:

- [x] The base branch is correct (no accidental merges)

## Reviewer Checklist

- Code readability and simplicity
- Follows best practices and coding standards
- Understandable and maintainable code

### CODE REVIEW

**Comment by akiomatic** [comment]
Could you revise the PR title as it does not follow our standard?
@yuki-92c

**Review by akiomatic — CHANGES_REQUESTED** [review]
Thank you for making changes, but could you revise the PR title too as it does not follow our standard?

**Comment by yuki-92c** [comment]
> Thank you for making changes, but could you revise the PR title too as it does not follow our standard?

thanks for the comment and edited the title.

File: docs/docs/modeling.md [hunk 1]
```diff
@@ -118,4 +118,28 @@ classDiagram
...
+
+- TaskId is the UUID
+
+### title
+
+- The owner can set a title as they prefer. This title can be modified at any time.
+
+Constraints:
+
+- It must be between 1 and 20 characters in length.
```
Comment (KaiKoide) [inline docs/docs/modeling.md:133]: I think you don't follow the notation rules in terms of the bullet point. Could you modify that?
https://scrapbox.io/tascurator/%F0%9F%8F%B7%EF%B8%8F_GitHub%E3%81%AE%E9%81%8B%E7%94%A8%E3%83%AB%E3%83%BC%E3%83%AB

File: docs/docs/modeling.md [hunk 2]
```diff
@@ -118,4 +118,28 @@ classDiagram
...
+
+### description
+
+- The owner can provide details for the task. This description can be modified at any time.
+
+Constraints:
+
+- The description must be between 10 and 120 characters.
```
Comment (KaiKoide) [inline docs/docs/modeling.md:141]: same as above.

**Review by KaiKoide — CHANGES_REQUESTED** [review]

**Review by Akikaze1119 — APPROVED** [review]
Thank you! LGTM👍

**Review by KaiKoide — APPROVED** [review]
Thank you for the modifications! LGTM 👍

**Review by HrRn — APPROVED** [review]
Thank you!

**Review by akiomatic — APPROVED** [review]
Thank you for fixing the title:)

**Pull request merged**